### PR TITLE
Editorial: `parseFloat`/`parseInt`: fix unintentional normative change from #1629

### PR DESCRIPTION
**Pull request by ljharb**
Thanks to @devsnek for discovering.

### CODE REVIEW

File: spec.html [hunk 1]
```diff
@@ -23708,7 +23708,8 @@ <h1>parseFloat ( _string_ )</h1>
       <p>The `parseFloat` function produces a Number value dictated by interpretation of the contents of the _string_ argument as a decimal literal.</p>
       <p>The `parseFloat` function is the <dfn>%parseFloat%</dfn> intrinsic object. When the `parseFloat` function is called with one argument _string_, the following steps are taken:</p>
       <emu-alg>
-        1. Let _trimmedString_ be ? TrimString(_string_, `"start"`).
+        1. Let _inputString_ be ? ToString(_string).
```
Comment (devsnek) [inline spec.html]:
```suggestion
        1. Let _inputString_ be ? ToString(_string_).
```

**Comment by devsnek** [comment]
BTW, the bug was with `parseX(undefined)` and `parseX(null)`, which should return `NaN`, not throw an error.

File: spec.html (hunk 1 above)
Comment (leobalter) [reply spec.html]: there is an extra occurrence at line 23731

**Comment by leobalter** [comment]
This is a nice caught and fix. I appreciate the work done here!

File: spec.html [hunk 2]
```diff
@@ -23727,7 +23728,8 @@ <h1>parseInt ( _string_, _radix_ )</h1>
       <p>The `parseInt` function produces an integer value dictated by interpretation of the contents of the _string_ argument according to the specified _radix_. Leading white space in _string_ is ignored. If _radix_ is *undefined* or 0, it is assumed to be 10 except when the number begins with the code unit pairs `0x` or `0X`, in which case a radix of 16 is assumed. If _radix_ is 16, the number may also optionally begin with the code unit pairs `0x` or `0X`.</p>
       <p>The `parseInt` function is the <dfn>%parseInt%</dfn> intrinsic object. When the `parseInt` function is called, the following steps are taken:</p>
       <emu-alg>
-        1. Let _S_ be ? TrimString(_string_, `"start"`).
+        1. Let _inputString_ be ? ToString(_string).
```
Comment (ljharb) [inline spec.html]:
```suggestion
        1. Let _inputString_ be ? ToString(_string_).
```

File: spec.html [hunk 3]
```diff
@@ -23708,7 +23708,8 @@ <h1>parseFloat ( _string_ )</h1>
       <p>The `parseFloat` function produces a Number value dictated by interpretation of the contents of the _string_ argument as a decimal literal.</p>
       <p>The `parseFloat` function is the <dfn>%parseFloat%</dfn> intrinsic object. When the `parseFloat` function is called with one argument _string_, the following steps are taken:</p>
       <emu-alg>
-        1. Let _trimmedString_ be ? TrimString(_string_, `"start"`).
+        1. Let _inputString_ be ? ToString(_string_).
+        1. Let _trimmedString_ be ? TrimString(_inputString_, `"start"`).
```
Comment (leobalter) [inline spec.html]: Nit: 

```suggestion
        1. Let _trimmedString_ be ! TrimString(_inputString_, `"start"`).
```

Sorry, I forgot to check this one before.

File: spec.html [hunk 4]
```diff
@@ -23727,7 +23728,8 @@ <h1>parseInt ( _string_, _radix_ )</h1>
       <p>The `parseInt` function produces an integer value dictated by interpretation of the contents of the _string_ argument according to the specified _radix_. Leading white space in _string_ is ignored. If _radix_ is *undefined* or 0, it is assumed to be 10 except when the number begins with the code unit pairs `0x` or `0X`, in which case a radix of 16 is assumed. If _radix_ is 16, the number may also optionally begin with the code unit pairs `0x` or `0X`.</p>
       <p>The `parseInt` function is the <dfn>%parseInt%</dfn> intrinsic object. When the `parseInt` function is called, the following steps are taken:</p>
       <emu-alg>
-        1. Let _S_ be ? TrimString(_string_, `"start"`).
+        1. Let _inputString_ be ? ToString(_string_).
+        1. Let _S_ be ? TrimString(_inputString_, `"start"`).
```
Comment (leobalter) [inline spec.html]:
```suggestion
        1. Let _S_ be ! TrimString(_inputString_, `"start"`).
```

**Comment by jmdyck** [comment]
(Note re title: the unintentional change was in #1602, not #1629.)

**Pull request merged**